testsuite/test_api.py

```diff
@@ -1,7 +1,10 @@
-from time import time, sleep
+import base64
 import subprocess
+from time import time, sleep
+
 import pytest
-from django.conf import settings # noqa
+from django.conf import settings  # noqa
```

Comment:
Why are we importing settings and then saying `noqa`? It seems pointless to import settings.